pages/common/openssl-genrsa.md

```diff
@@ -0,0 +1,15 @@
+# openssl genrsa
+
+> OpenSSL command to generate RSA private keys.
```

Comment:
How about a more information link?

```suggestion
> OpenSSL command to generate RSA private keys.
> More information: <https://www.openssl.org/docs/man1.0.2/man1/genrsa.html>.
```

Just for future reference although this URL has a specific version in it, this is probably fine because according to  [this page](https://www.openssl.org/docs/manmaster/man1/openssl.html) the `genrsa` subcommand has been superseded by the `genpkey` subcommand apparently.